astra/src/test/java/com/slack/astra/chunk/IndexingChunkImplTest.java

```diff
@@ -499,16 +499,7 @@ public void tearDown() throws IOException, TimeoutException {
 
     @Test
     public void testAddInvalidMessagesToChunk() {
-      Trace.Span invalidSpan =
-          Trace.Span.newBuilder()
-              .setId(ByteString.copyFromUtf8("1"))
-              .addTags(
-                  Trace.KeyValue.newBuilder()
-                      .setVInt32(123)
-                      .setKey(LogMessage.ReservedField.MESSAGE.fieldName)
-                      .setFieldType(Schema.SchemaFieldType.INTEGER)
-                      .build())
-              .build();
+      Trace.Span invalidSpan = Trace.Span.newBuilder().build();
```

Comment:
old way of creating invalid span doesn't work. Earlier we took advantage of the fact `message` was predefined as a string but we said in the span it's Integer.

So throw error

New way is just to create empty span - we fail without ID.